### PR TITLE
Project: Backgrounds and Gradients 

### PR DESCRIPTION
**Pull request by babs257**

### CODE REVIEW

**Comment by wathigo** [comment]
## Hello @babs257 
**Good work so far**, 
just a couple of things required to mark this as complete.

1. Add the config files for linting in your project. See [this](https://github.com/microverseinc/linters-config#linters-config) for more info. You have integrated the repository with stickler-ci and added an empty `.stickler.yml` but you need add the required configurations in the file to make this work. You should correct all offenses pointed out by the linter check.

2. Add a meaningful name for the Pull Request. Remember to include a description of the tasks you have completed in the Pull Request.

3. The background image you have used contains both the bottom four elements and 
the navigation bar. Include the four bottom elements with background images. 
They should be links. Hint: _use flex to style them_.

4. Make sure the background image used does not contain a navigation bar. Am not sure
if the styles for the navigation bar are coming from your styles or the background image. 

5. Make use of semantic html5 elements. Make sure you page contains 
[semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp). Include `header` element containing the nav and `main` element containing the main part of the page. 

6. Make sure you don't use the original screenshot to bypass the requirements.

Please do necessary changes then submit another request for review.

Feel free to reach out via slack using or [twitter](https://twitter.com/simon_wathigo) if you need further clarifications.

_I have noticed that you have used a background image for the project.
Please use a background image only when necessary. The goal of this project
is to test you ability to style a page and should contain the page structure, i.e 
`header`, `main`, `footer`. Use a background image only for specific elements of the page
that have a background image._

**Comment by wathigo** [comment]
## Hello @babs257 
**Good work so far**, 
just a couple of things required to mark this as complete.

1. Add the config files for linting in your project. See [this](https://github.com/microverseinc/linters-config#linters-config) for more info. You have integrated the repository with stickler-ci and added an empty `.stickler.yml` but you need add the required configurations in the file to make this work. You should correct all offenses pointed out by the linter check.

2. Add a meaningful name for the Pull Request. Remember to include a description of the tasks you have completed in the Pull Request.

3. The background image you have used contains both the bottom four elements and 
the navigation bar. Include the four bottom elements with background images. 
They should be links. Hint: _use flex to style them_.

4. Make sure the background image used does not contain a navigation bar. Am not sure
if the styles for the navigation bar are coming from your styles or the background image. 

5. Make use of semantic html5 elements. Make sure you page contains 
[semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp). Include `header` element containing the nav and `main` element containing the main part of the page. 

6. Make sure you don't use the original screenshot to bypass the requirements.

Please do necessary changes then submit another request for review.

Feel free to reach out via slack using or [twitter](https://twitter.com/simon_wathigo) if you need further clarifications.

_I have noticed that you have used a background image for the project.
Please use a background image only when necessary. The goal of this project
is to test you ability to style a page and should contain the page structure, i.e 
`header`, `main`, `footer`. Use a background image only for specific elements of the page
that have a background image._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+			<div id="gn-store"><a class="nav-icons" href="#"><span>Store</span></a></div>
+			<div id="gn-mac"><a class="nav-icons" href="#"><span>Mac</span></a></div>
+			<div id="gn-ipod"><a class="nav-icons" href="#"><span>iPod</span></a></div>
+			<div id="gn-iphone"><a class="nav-icons" href="#"><span>iPhone</span></a></div>
+			<div id="gn-ipad"><a class="nav-icons" href="#"><span>iPad</span></a></div>
+			<div id="gn-itunes"><a class="nav-icons" href="#"><span>iTunes</span></a></div>
+			<div id="gn-support"><a class="nav-icons" href="#"><span>Support</span></a></div>
+			</div>
+			<div id="global-search">
+				<form id="search-form" action="#" method="post"><i class="fa fa-search" style="width:  15px; height: 14px; margin-left: 6px; margin-top: 3px;"></i>
```
Comment (wathigo) [inline index.html]: [REQUIRED] Remove inline styles Include all your styles in the external CSS file

File: styles/style.css [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+}
+
+#content-2 {
+	background-color: white;
+    border-bottom: 1px lightgray solid;
+    position: relative;
+    margin: auto;
+    height: 720px;
+    width: 1024px;
+    background-image: url("https://archive.fo/UW4oR/bf13112842d2dd0160466e36cfafac05d80eeb41/scr.png");
```
Comment (wathigo) [inline styles/style.css]: [REQUIRED] Don't use a a background image to bypass requirements.

**Review by wathigo — CHANGES_REQUESTED** [review]
## Hello @babs257 
**Good work so far**, 
just a couple of things required to mark this as complete.

1. Add the config files for linting in your project. See [this](https://github.com/microverseinc/linters-config#linters-config) for more info. You have integrated the repository with stickler-ci and added an empty `.stickler.yml` but you need add the required configurations in the file to make this work. You should correct all offenses pointed out by the linter check.

2. Add a meaningful name for the Pull Request. Remember to include a description of the tasks you have completed in the Pull Request.

3. The background image you have used contains both the bottom four elements and 
the navigation bar. Include the four bottom elements with background images. 
They should be links. Hint: _use flex to style them_.

4. Make sure the background image used does not contain a navigation bar. Am not sure
if the styles for the navigation bar are coming from your styles or the background image. 

5. Make use of semantic html5 elements. Make sure you page contains 
[semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp). Include `header` element containing the nav and `main` element containing the main part of the page. 

6. Make sure you don't use the original screenshot to bypass the requirements.

7. Remove inline CSS in your HTML code.

**Please do necessary changes then submit another request for review.**

Feel free to reach out via slack using or [twitter](https://twitter.com/simon_wathigo) if you need further clarifications.

_I have noticed that you have used a background image for the project.
Please use a background image only when necessary. The goal of this project
is to test you ability to style a page and should contain the page structure, i.e 
`header`, `main`, `footer`. Use a background image only for specific elements of the page
that have a background image._

File: styles/style.css [hunk 3]
```diff
@@ -0,0 +1,392 @@
...
+  border-style: none;
+  border-color: white;
+  font-style: normal;
+}
+
+#footer-image {
+  display: block;
+  justify-content: center;
+  text-align: center;
+  position: relative;
```
Comment (samgaco) [inline styles/style.css]: The section for **#footer-image** should be aligned using display flex or floats.

**Review by samgaco — CHANGES_REQUESTED** [review]
Hi @babs257 :high_brightness: 

Now is looking better :100: , I can see the effort you are putting in. Some more changes before we are able to give you the approval:

--------------

### Required:

1. Solve all the linter checks that have failed

> failed6

2. The four boxes at the bottom should be aligned with float OR flex

### So, please just work on these 2 :crossed_fingers: points and submit back for another code review. Regards!
--------------

### Optional:

* Read more about  HTML semantic elements an implement a few more, like `<aside>` , `<section>` and `<article>` 
* Try to make the styling of the site looking closer to that of the original website. Right  now is looking a bit off and trying to make it look similar would force you to keep improving as a developer 

-----------------------

Moving forward have this into account :arrow_down: 

* **[For your next code review requests]**  Add a short summary of changes in Pull Request description


-------------

Summary: You have two required points 1. and 2. and two optional points right after.
* ###### STATUS OF THE PROJECT: REQUESTED CHANGES :stop_sign:

File: README.md [hunk 4]
```diff
@@ -1 +1,5 @@
-# apple-page
\ No newline at end of file
+# apple-page
+
+This project consists of building a webpage using images as a background and adding gradients to elements. The goal is to make a clone of old an old version of the Apple website webpage.
+
+In this project you’ll clone the website of one of the most design-forward companies in the world. It’s not a very complex site but it uses background photos well and the navbar at the top contains a gradient that you should be able to mimic.
```
Comment (bolah2009) [inline README.md]: Let's make the `README` file more descriptive, details like what was done in the projects, what was learned, link to the assignment, clone and source page, screenshots of the clone page can make README look more professional.
 > Remember about descriptive README file.

**Review by bolah2009 — CHANGES_REQUESTED** [review]
# Hi @babs257  :wave: 

### Nice job so far, good job with the navbar gradients too. :smile: 

#### Let's make [the clone](https://raw.githack.com/babs257/apple-page/dc1d962a1718cf50d150c6e042509817388a7a40/index.html) look like the [source page](https://web.archive.org/web/20140301004610/http://www.apple.com/) please.

The main objective of the project is using CSS Backgrounds effectively, the main background image should cover the whole page, remove the border from the page and add hover effects to the navbar accordingly. :relaxed: 

| Current | Preferred |
|:----:|:-----:|
|image | image|

Other things to note:
- Fix [HTML Validation errors/warnings](https://validator.w3.org/nu/?doc=https%3A%2F%2Fraw.githack.com%2Fbabs257%2Fapple-page%2Fdc1d962a1718cf50d150c6e042509817388a7a40%2Findex.html)
  > Validate your HTML markup with https://validator.w3.org/#validate_by_upload.
- Make `README` file more descriptive
  > Remember about descriptive README file.
- Let's make this Pull Request look more professional. [See below](https://github.com/babs257/apple-page/pull/1#pullrequestreview-291477910)
  - [**NEXT PROJECT**]: Use meaningful commits messages. [Learn more here](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)
  - [:heavy_check_mark: **Done**] Use meaningful names for Pull Requests.
  - [**REQUIRED**] Add a short summary of changes in Pull Request description.
- [NEXT PROJECTS]: For next (HTML/CSS) project, please follow `GitHub flow` [guidelines](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/). What is missing here is a descriptive branch name instead of `feature-branch`, think of feature branch as a branch name that is descriptive enough to have an idea of what the branch is used for.

#### :recycle:  Make these changes and submit another code review request. :+1: 

#### Feel free to reach me on slack if you having issues implementing these features. :+1: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Review by bolah2009 — COMMENTED** [review]
# Update PR description

  update-PR-description-html

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello @babs257  

**Good work so far 👍 but you still need to fix the following issues before I can give you a thumb's up ⬇️** 

1. Align the 4 bottom elements to the center just like the original page.

2. Fix markup validation errors on [here](https://validator.w3.org/).

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Review by stratospherique — APPROVED** [review]
### Great work :clap: . You made all the necessary changes :100: . The project is approved :heavy_check_mark: and you can merge to master :+1: :+1:

**Pull request merged**